### ISSUE TITLE
Having a "game/data/" directory causes issues with Ren'Py?

### ISSUE THREAD

**Issue by micka190** [renpy/renpy#5341]
We've been trying to do some refactoring on our project, and we ended-up trying to create a `game/data/` directory. We put some files that generate in-game data in there.

Problem is, as soon as we create a `game/data/` directory, the game no longer launches. For some reason, trying to launch the game fails with the following error:

```log
I'm sorry, but an uncaught exception occurred.

While running game code:
  File "renpy/common/00start.rpy", line 288, in script
    python:
ScriptError: could not find label 'start'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "C:\dev\renpy\renpy-8.1.3-sdk\renpy\bootstrap.py", line 359, in bootstrap
    renpy.main.main()
  File "C:\dev\renpy\renpy-8.1.3-sdk\renpy\main.py", line 635, in main
    run(restart)
  File "C:\dev\renpy\renpy-8.1.3-sdk\renpy\main.py", line 145, in run
    renpy.execution.run_context(True)
  File "C:\dev\renpy\renpy-8.1.3-sdk\renpy\execution.py", line 955, in run_context
    context.run()
  File "renpy/common/00start.rpy", line 288, in script
    python:
  File "C:\dev\renpy\renpy-8.1.3-sdk\renpy\script.py", line 1103, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'start'.

Windows-10-10.0.19045 AMD64
Ren'Py 8.2.0.24012702
 
Mon Feb  5 14:51:41 2024
```

Additionally, after it crashes, we can see that Ren'Py has created some directories *inside* the `game/data/` directory along with some files:
- `game/data/saves/`:
  - `sync/persistent`
  - `persistent`
- `game/data/cache/`:
  - `bytecode-39.rpyb`
  - `py3analysis.rpyb`
  - `screens.rpyb`
  - `shaders.txt`

Renaming the `game/data/` directory to anything else fixes the issue and Ren'Py does *not* appear to create a `game/data/` directory of its own.

Is this supposed to be a reserved directory name in Ren'Py projects or something? Or is this a bug?  If this is some documented feature, I can't find it for the life of me.

**Comment by qTich** [comment]
> Is this supposed to be a reserved directory name in Ren'Py projects or something? Or is this a bug? If this is some documented feature, I can't find it for the life of me.

https://github.com/renpy/renpy/blob/master/renpy.py#L61

**Comment by micka190** [comment]
Ah, there we go! That explains it. I'm assuming it does that for legacy reasons?

**Comment by renpytom** [comment]
I think that something is going wrong here, as it shouldn't look for it twice. I'll figure it out.

**Comment by renpytom** [comment]
Hm... I can't repeat it. 

When you say game/data, do you mean:

1. project/game/data
2. project/data

? Basically, what do the files in the bad project look like?

**Comment by micka190** [comment]
I mean `project/game/data/`, yeah.

We've got the `project/` directory where the `game/` directory is, alongisde things like the `icon.ico` file.
Then, inside the `game/` directory we've got the standard stuff:

- `save/`
- `audio/`
- `gui/`
- `tl/`
- etc.

It occurs if we put the `data/` directory there.

**Comment by renpytom** [comment]
Can you share the project with me? I can't repeat it. 

My email is renpytom@gmail.com.

**Comment by micka190** [comment]
Good news, I found the issue while trying to make a minimal reproducible example.

 We've been using a VS Code Task that runs `renpy.exe` so we can quickly launch the project without having to go through the launcher. Your previous comment got me thinking. We had it set to launch `project/game/`. I'm guessing Ren'Py found `data/` first when we did that. I changed the Task to launch `project/` instead and it no longer crashes with the `data/` folder.

Seems like it was user error and not an actual bug, my bad 😅
